frame/elections-phragmen/src/benchmarking.rs

```diff
@@ -0,0 +1,453 @@
+// Copyright 2020 Parity Technologies (UK) Ltd.
+// This file is part of Substrate.
+
+// Substrate is free software: you can redistribute it and/or modify
+// it under the terms of the GNU General Public License as published by
+// the Free Software Foundation, either version 3 of the License, or
+// (at your option) any later version.
+
+// Substrate is distributed in the hope that it will be useful,
+// but WITHOUT ANY WARRANTY; without even the implied warranty of
+// MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+// GNU General Public License for more details.
+
+// You should have received a copy of the GNU General Public License
+// along with Substrate.  If not, see <http://www.gnu.org/licenses/>.
+
+//! Elections-Phragmen pallet benchmarking.
+
+#![cfg(feature = "runtime-benchmarks")]
+
+use super::*;
+
+use codec::{Encode, Decode};
+use frame_system::RawOrigin;
+use sp_io::hashing::blake2_256;
+use frame_benchmarking::benchmarks;
+use sp_runtime::traits::Bounded;
+use frame_support::traits::OnInitialize;
+
+use crate::Module as Elections;
+
+const SEED: u32 = 0;
+const BALANCE_FACTOR: u32 = 250;
+const MAX_LOCKS: u32 = 20;
+const MAX_VOTERS: u32 = 50_000;
+const MAX_CANDIDATES: u32 = 1000;
+
+type Lookup<T> = <<T as frame_system::Trait>::Lookup as StaticLookup>::Source;
+
+/// grab a new account
+fn account<T: Trait>(name: &'static str, index: u32) -> T::AccountId {
```

Comment:
Please use the `account` function provided by the runtime benchmarking library, and specifically if it is being used to generate the "caller" account, use `account("caller", 0, SEED)` so we can easily track the account